src/components/filters/PbDateRangeFilters.vue

```diff
@@ -167,13 +167,25 @@ export default {
         return;
       }
 
+      this.sendClickInsight();
+
       return this.$router.replace({
         query: {
           ...query,
           [this.alias]: queryString
         }
       });
     },
+    sendClickInsight() {
+      let dates = [
+        this.dates.start ? `${this.alias}:>=${this.dates.start}` : null,
+        this.dates.to ? `${this.alias}:<=${this.dates.to}` : null
+      ];
+
+      this.sendFilterAppliedInsight(
+        dates.filter(d => d)
```

Comment:
@fdalcin is the filter necessary?

Reply:
When filtering only the start date, for example, the `dates` array will be like this:

```js
[
  'updated:>=2021-06-29',
  null
]
```

I'm applying this `filter` to remove `null` values.

Reply:
got it, thanks!